com.microsoft.mrtk.graphicstools.unity/Samples~/MeshOutlines/Prefabs/leCat_01.prefab

```diff
@@ -0,0 +1,1698 @@
+%YAML 1.1
```

Comment:
nit: I wonder if we could delete this prefab since the model is only used in one scene at the moment? Normally I like creating prefabs like this but to keep the sample package small this would remove a file.